docroot/modules/custom/mass_alerts/templates/mass-alerts-block.html.twig

```diff
@@ -1,4 +1,5 @@
 {% if path %}
   {{ attach_library('mass_alerts/behaviors') }}
-  <div class="mass-alerts-block" data-alerts-path="{{ path }}"></div>
+  <div class="mass-alerts-block"></div>
+  <script type="text/javascript">alerts("{{ path }}", "{{ node_type }}", document.currentScript.previousElementSibling)</script>
```

Comment:
is there a reason why the data attribute approach will no longer work?

Reply:
Because the alerts function was already receiving parameters, hence we can send another parameter instead of looking inside the alerts script and trying to determine the path based on an external block.